docs/tutorial/2-tilt-up.md

```diff
@@ -36,48 +36,47 @@ tilt up --port=3366
 You should see output similar to the following in your terminal:
 ![Running tilt up in a Terminal window shows "Tilt started on http://localhost:3366/" message](/assets/img/tutorial/tilt-up-cli.gif)
 
-Press `Spacebar` while your terminal is active and Tilt will launch your default browser.
-Alternatively, navigate to [http://localhost:3366/]() yourself.
+Hit `Spacebar` while your terminal is active, and Tilt will launch your default browser.
+(Or just navigate to [http://localhost:3366/]())
 
-In the next section, we'll explain the Tilt UI: but first, let's dissect what's happening in the background.
+In the next section, we'll explain the Tilt UI. But first, let's dissect what's happening in the background.
```

Comment:
I feel like you have the same relationship to `:` that I have to `—`...pushing the boundaries of using it whenever I can to simulate the cadence of speech! But here the colon was a bit less logical.

Reply:
![maybe...](https://c.tenor.com/YTPbiFYWAsEAAAAd/excited-happy.gif)